### ISSUE TITLE
There was an error loading OptiFabric!

### ISSUE THREAD

**Issue by faceless333** [Chocohead/OptiFabric#1502]
There was an error loading OptiFabric!
Failed to load OptiFine, please report this!
net/fabricmc/tinyremapper/IMappingProvider

**Comment by Dracconis** [comment]
Try using Fabric Loader 0.14.25.

**Comment by Crosspro2030** [comment]
the same problem
only the addition of Fabric Loader has not changed anything

**Comment by Dracconis** [comment]
> the same problem only the addition of Fabric Loader has not changed anything

Then instead of hijacking someone elses ticket open your own ticket and provide the information

**Comment by koolkidjimmy** [comment]
@Crosspro2030

Check your Fabric API, sometimes that can interfere with Optifine (Sorta rare, but it happens).

If that didn't work, check your version of OptiFabric (Latest version is 1.13.6 I think, make sure you are using that)

Since you said Fabric-Loader isn't the problem, it might be if you don't have the latest version of OptiFabric, if you don't want to switch OptiFabric version, I recommend trying 0.14.20 to see if that fixes the issue.

Hope it helped!

**Comment by Dracconis** [comment]
> @Crosspro2030
> 
> Check your Fabric API, sometimes that can interfere with Optifine (Sorta rare, but it happens).
> 
> If that didn't work, check your version of OptiFabric (Latest version is 1.13.6 I think, make sure you are using that)
> 
> Since you said Fabric-Loader isn't the problem, it might be if you don't have the latest version of OptiFabric, if you don't want to switch OptiFabric version, I recommend trying 0.14.20 to see if that fixes the issue.
> 
> Hope it helped!

You are providing information that isnt required and can be confusing. The answer I provided is the solution to their problems.

**Comment by koolkidjimmy** [comment]
> > @Crosspro2030
> > Check your Fabric API, sometimes that can interfere with Optifine (Sorta rare, but it happens).
> > If that didn't work, check your version of OptiFabric (Latest version is 1.13.6 I think, make sure you are using that)
> > Since you said Fabric-Loader isn't the problem, it might be if you don't have the latest version of OptiFabric, if you don't want to switch OptiFabric version, I recommend trying 0.14.20 to see if that fixes the issue.
> > Hope it helped!
> 
> You are providing information that isnt required and can be confusing. The answer I provided is the solution to their problems.

Yup it was a few weeks ago and your reply seemed credible, don't know why I commented, that's my bad.
